src/components/filePreview/filePreview.tsx

```diff
@@ -24,9 +28,23 @@ export default function FilePreview(
       variant="outlined"
       sx={{ boxShadow: theme.shadows[1] }}
     >
-      <Typography variant="subtitle2" data-cy="file-name">
-        {shortenString(props.file.name, maximumFileNameLength)}
-      </Typography>
+      <Box className="rustic-file-preview-header">
+        <Typography variant="subtitle2" data-cy="file-name">
+          {shortenString(props.file.name, maximumFileNameLength)}
+        </Typography>
+        {props.file.url && (
```

Comment:
I'm wondering if you want to pass download as a children component to filePreview. It depends on if we want to support download in the Input as well

Reply:
Are there any scenarios where you would need to download if you're the one uploading the file?

Reply:
Personally, I think it's not needed in the Input component. If that's the case, maybe you should pass the download logic as children component so that it won't be used in the Input.